integration/mdc-migration/verify-golden.ts

```diff
@@ -0,0 +1,190 @@
+import {promises as fsPromises, Stats} from 'fs';
+import path from 'path';
+
+// Expected usage:
```

Comment:
Nit: I know this is quite obvious, but maybe a small fileoverview comment would be nice to match up with how we author most other scripts?